components/automate-cli/cmd/chef-automate/backup.go

```diff
@@ -785,17 +797,43 @@ func runDeleteBackupCmd(cmd *cobra.Command, args []string) error {
 			"Must specify a backup",
 		)
 	}
+	var location string
+	start := 0
+	var newArgs []string
+	var backup []*api.BackupTask
+
+	if strings.Contains(args[0], "/") || strings.Contains(args[0], "\\") {
+		location = args[0]
+		start = 1
+	}
+
+	backup, err := getBackupTask(location)
+	if err != nil {
+		return errors.Errorf("Unable to fetch the backup list with error: %v", err.Error())
+	}
+
+	backupMap := getBackupMapFromBackupList(backup)
 
 	ids, err := idsToBackupTasks(args)
 	if err != nil {
 		return err
+	}
 
+	for i := start; i < len(args); i++ {
+		backupState, ok := backupMap[args[i]]
+		if !ok {
+			writer.Failf("The backup Id %s is either removed or typed incorrect.", args[i])
+		} else if !getBackupStatusAsCompletedOrFailed(backupState) {
+			writer.Failf("The backup Id %s is still in progress", args[i])
```

Comment:
We should not mention specifically in progress. We can mention `The backup ID %s cannot be deleted currently.` We can get suggestions from Doc team